README.md

```diff
@@ -412,10 +412,6 @@ parameters:
             output: match-10.0
             type: add_regex_if
             parameters: 10.0.*
-          - input: "{{.srcIP}},{{.srcPort}},{{.dstIP}},{{.dstPort}},{{.protocol}}"
-            output: isNewFlow
-            type: conn_tracking
-            parameters: "1"
 ```
 
 The first rule `add_subnet` generates a new field named `srcSubnet` with the 
```

Comment:
Will there be a new section on describing the connection tracking?

Reply:
I documented the new connection tracking in #291